spec/lib/local-links-manager/import/local_custodian_codes_importer_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe LocalLinksManager::Import::LocalCustodianCodesImporter do
+  describe "#import_from_csv" do
+    context "local custodian codes for Local Authorities in Great Britain" do
+      let!(:local_authority_1) { create(:local_authority, slug: "leicestershire", local_custodian_code: nil) }
+      let!(:local_authority_2) { create(:local_authority, slug: "surrey-heath", local_custodian_code: nil) }
+      let!(:local_authority_3) { create(:local_authority, slug: "stockton-on-tees", local_custodian_code: nil) }
+
+      it "populates code with value in different formats" do
+        LocalLinksManager::Import::LocalCustodianCodesImporter.new.import_from_csv("spec/lib/local-links-manager/import/fixtures/addressbase-local-custodian-codes-test.csv")
+
+        local_authority_1.reload
+        local_authority_2.reload
+        local_authority_3.reload
+        expect(local_authority_1.local_custodian_code).to eq "2460"
+        expect(local_authority_2.local_custodian_code).to eq "3640"
+        expect(local_authority_3.local_custodian_code).to eq "738"
+      end
+    end
+
+    context "local custodian codes for Local Authorities in Northern Ireland" do
+      let!(:local_authority_1) { create(:local_authority, slug: "belfast", local_custodian_code: nil) }
+
+      it "populates code with value" do
```

Comment:
As a general rule, we want to avoid using `context` when it only wraps around a single test. It's a sign that it's been slightly over-engineered. These tests would read more clearly as something like:

```ruby
it "extracts local custodian codes from CSVs containing Northern Ireland Local Authorities" do
  local_authority = create(:local_authority, slug: "belfast", local_custodian_code: nil)
  described_class.new.import_from_csv("spec/lib/local-links-manager/import/fixtures/addressbase-local-custodian-code-Northern-Ireland-test.csv")
  expect(local_authority_1.reload.local_custodian_code).to eq "8132"
end
```